sample.env

```diff
@@ -77,13 +77,6 @@ CSM_EMAILS_TO_SEND_TO=
 CSM_TARGET_SMTP_SERVER_ADDRESS=
 CSM_TARGET_SMTP_SERVER_PORT=
 
-# Email configuration for sending firmware-manager failure emails
-SMTP_EMAIL=
-SMTP_USERNAME=
-SMTP_PASSWORD=
-FW_EMAIL_RECIPIENTS=
-SMTP_SERVER_IP=
-

```

Comment:
Why was this removed?

Reply:
good catch, sorry! I replaced it with another sample... Fixed this now